src/storage.js

```diff
@@ -0,0 +1,46 @@
+import Store from './addremove.js';
+
+class Storage {
+  static getTodos() {
+    let todo;
+    if (localStorage.getItem('tasks') === null) {
+      todo = [];
+    } else {
+      todo = JSON.parse(localStorage.getItem('tasks'));
+    }
+
+    return todo;
+  }
+
+  static addTodos(todo) {
+    localStorage.setItem('tasks', JSON.stringify(todo));
+  }
+
+  static checkCompleted(id) {
+    const completed = Store.getTasks();
+    completed.forEach((task) => {
+      if (task.index == id) {
```

Comment:
line 22 and line 37, kindly use strictly equal to avoid conversion of types during comparison which could lead to the code not performing as we want